### DISCUSSION TITLE
PrimeNG reorderable table example broken

### DISCUSSION THREAD

**Discussion by giorgio79** [primefaces/community#311] [PrimeNG] [answered]
Hello,

I notice the stackblitz example for reorderable primeng table is broken
https://primeng.org/table#reorder
https://stackblitz.com/edit/primeng-tablereorder-demo?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.ts

![image](https://github.com/primefaces/community/assets/283347/87aadc50-dabd-4576-8861-bb7b0ec108ca)

**Comment by sonicwong** [comment, marked as answer]
It is working. resize window to desktop size and it will show as a row.
